### PR TITLE
#148263203 get all approved contributions

### PR DESCRIPTION
**Pull request by andela-uibrahim**
## What does this PR do?
This PR adds the functionality for users to get all approved contributions

## Description of Task to be completed?
create endpoint to get all approved contributions
Write tests for the new feature

## How should this be manually tested?
clone the repository
checkout to the branch
run flask db upgrade
the navigate to the endpoint and include status params "http://localhost:5000/api/v1/contributions?status=approved" 

## What are the relevant pivotal tracker stories?

#148263203

### CODE REVIEW

File: api/views/contributions.py [hunk 1]
```diff
@@ -27,12 +27,16 @@ def get(self):
...
         search_term = request.args.get('q')
-
+        status = request.args.get('status')
+    
         _contributions = Contribution.query.order_by(
                     Contribution.created_at.desc())
         if search_term:
             _contributions = _contributions.filter(
                 Contribution.description.ilike('%{}%'.format(search_term)))
+        if status == "approved":
```
Comment (LarryWachira) [inline api/views/contributions.py]: Perhaps you also allow filtering by pending as well...

Comment (LarryWachira) [reply api/views/contributions.py]: @andela-uibrahim You forgot to reply to my comment above...

**Comment by andela-oowonikoko** [comment]
@andela-uibrahim Please rebase so I can review

File: test/api/views/test_contributions.py [hunk 2]
```diff
@@ -144,10 +144,11 @@ def test_search_for_contributions_with_query(self):
 
         response_json = json.loads(response.data)
         response_contributions = response_json["data"]
+        print(response_contributions)
```
Comment (andela-oowonikoko) [inline test/api/views/test_contributions.py]: Please remove this print statement

**Comment by andela-oowonikoko** [comment]
LGTM

**Comment by andela-oowonikoko** [comment]
@andela-uibrahim Please rebase your branch :thank_you:

**Comment by Andela-ddiei** [comment]
LGTM.

**Pull request merged**